task/01-strings-tasks.js

```diff
@@ -201,7 +202,18 @@ function extractEmails(str) {
  *
  */
 function getRectangleString(width, height) {
-    throw new Error('Not implemented');
+    var top = '┌'.concat('─'.repeat(width - 2)).concat('┐');
```

Comment:
Please, try to use only this list of statements and methods

1. const
2. concat
3. repeat
4. return :)